lambdas/handlers/authoriser_handler.py

```diff
@@ -0,0 +1,221 @@
+"""
+This code has been modified from AWS blueprint. Below is the original license:
+
+Copyright 2015-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License").
+You may not use this file except in compliance with the License. A copy of the License is located at
+     http://aws.amazon.com/apache2.0/
+or in the "license" file accompanying this file.
+This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+"""
+
+import logging
+import os
+import re
+
+import boto3
+import botocore
+import jwt
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def lambda_handler(event, context):
+    user = "Unauthorised"
+    ssm_public_key_parameter_name = os.environ["SSM_PARAM_JWT_TOKEN_PUBLIC_KEY"]
```

Comment:
How are we planning to set the os.env on the aws lambda instance?

Reply:
In a branch for subtask 165, we added this env var to the lambda by terraform : https://github.com/nhsconnect/national-document-repository-infrastructure/blob/prmdr_165_subtask/infrastructure/lambda-authoriser.tf